Monero/0.17.1.7/linuxamd64.Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN adduser --system --group --disabled-password monero && \
 VOLUME /home/monero/.bitmonero
 VOLUME /wallet
 
+# Download and add block.txt to block known-malicious hosts
+ADD https://gui.xmr.pm/files/block.txt /home/monero/.bitmonero/
```

Comment:
Needs to be:
`ADD https://gui.xmr.pm/files/block.txt /home/monero/block.txt
`

Reply:
Fixed.